src/widgets/banner/patient-banner.component.tsx

```diff
@@ -61,33 +63,51 @@ export default function PatientBanner() {
             </div>
             <div className={styles.patientInfo}>
               <div className={(styles.row, styles.nameRow)}>
-                <span className={styles.patientName}>{getPatientNames()}</span>
-                {hasActiveVisit && (
-                  <TooltipDefination
-                    align="end"
-                    tooltipText={
-                      <div className={styles.tooltipPadding}>
-                        <h6 style={{ marginBottom: "0.5em" }}>
-                          {currentVisit &&
-                            currentVisit.visitType &&
-                            currentVisit.visitType.name}
-                        </h6>
-                        <span>
-                          <span className={styles.tooltipSmalltext}>
-                            Started:{" "}
-                          </span>
+                <div>
+                  <span className={styles.patientName}>
+                    {getPatientNames()}
+                  </span>
+                  {hasActiveVisit && (
+                    <TooltipDefination
+                      align="end"
+                      tooltipText={
+                        <div className={styles.tooltipPadding}>
+                          <h6 style={{ marginBottom: "0.5em" }}>
+                            {currentVisit &&
+                              currentVisit.visitType &&
+                              currentVisit.visitType.name}
+                          </h6>
                           <span>
-                            {dayjs(
-                              currentVisit && currentVisit.startDatetime
-                            ).format("DD - MMM - YYYY @ HH:mm")}
+                            <span className={styles.tooltipSmalltext}>
+                              Started:{" "}
+                            </span>
+                            <span>
+                              {dayjs(
+                                currentVisit && currentVisit.startDatetime
+                              ).format("DD - MMM - YYYY @ HH:mm")}
```

Comment:
How are we handling date rendering, @denniskigen or @donaldkibet ?

Reply:
I saw somewhere already using the dayjs in the code and then checked it's documentation before implementing.

Reply:
Yeah, we do use dayjs, and you are using it in a way that is technically correct. However, we need dates to be localized correctly. So no static formatting.

Reply:
Oh wait... I'm looking at the patient-chart-widgets code and it looks like we have really not solved this problem. Maybe just use `DD-MMM-YYYY hh:mm A`. At some point we'll need to get our date rendering straightened out.